test/evergreen.yml

```diff
@@ -2978,6 +2978,11 @@ tasks:
           remote_file: wiredtiger/${build_variant}/${revision}/${dependent_task}_${build_id}-${execution}/full_coverage_report.json
           bucket: build_external
           local_file: wiredtiger/coverage_report/full_coverage_report.json
+      - command: github.generate_token
+        params:
+          expansion_name: github_token
+          permissions:
+            pull_requests: write
```

Comment:
I'm wondering where this permission name "pul_requests" is defined. One example in the [documentation](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Github-Integrations#dynamic-github-access-tokens) reads: 
```
{
    pull-request: read
    actions: write
}
```

Reply:
The docs for the [command](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Commands#githubgenerate_token) link to github [docs](https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#create-an-installation-access-token-for-an-app) which define the options in the `properties of permissions` expandable thing.

>pull_requests string
>The level of permission to grant the access token for pull requests and related comments, assignees, labels, milestones, and merges.
>
>Can be one of: read, write

Reply:
Ah I see - needs to expand the "properties of permissions" to see all available property names on the documentation page.